src/screens/FeedScreen/Footer.js

```diff
@@ -103,12 +103,12 @@ const styles = StyleSheet.create({
 
 const handleTextCountColor = (totalVote) => {
   if (totalVote > 0) {
-    return '#00ADB5';
+    return COLORS.anon_primary;
   }
   if (totalVote < 0) {
-    return '#FF2E63';
+    return COLORS.redalert;
   }
-  return '#C4C4C4';
+  return COLORS.lightgrey;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for maintainability and consistency. By using the `COLORS` object, it's easier to manage color themes across the application. However, I would suggest adding a default case in the function `handleTextCountColor` to handle any unexpected values of `totalVote`. This will improve error handling and make the code more robust.

```diff
 const handleTextCountColor = (totalVote) => {
   if (totalVote > 0) {
     return COLORS.anon_primary;
   }
   if (totalVote < 0) {
     return COLORS.redalert;
   }
+  if (totalVote === 0) {
     return COLORS.lightgrey;
+  }
+  throw new Error(`Unexpected totalVote value: ${totalVote}`);
 };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->